### PR TITLE
[ansible] Install docker for Ubuntu on RISC-V

### PR DESCRIPTION
**Pull request by luhenry**
##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/main.yml [hunk 1]
```diff
@@ -45,7 +46,19 @@
...
-      when: ((ansible_distribution == "RedHat" or ansible_distribution == "CentOS") and (ansible_distribution_major_version >= "7")) or (ansible_distribution == "Ubuntu" or ansible_distribution == "Debian")
+      when:
+        - ((ansible_distribution == "RedHat" or ansible_distribution == "CentOS") and (ansible_distribution_major_version >= "7")) or (ansible_distribution == "Ubuntu" or ansible_distribution == "Debian")
+        - ansible_architecture != "riscv64"
+
+    - name: Install for Ubuntu on RISC-V
+      package:
+        name:
+          - docker.io
+        state: latest   # TODO: Package installs should not use latest
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/main.yml:57]:
```suggestion
        state: latest  # TODO: Package installs should not use latest
```

**Review by karianna — CHANGES_REQUESTED** [review]
Probably need to fix that TODO for security/stability reasons.

**Review by gdams — APPROVED** [review]

**Review by karianna — APPROVED** [review]
LGTM

**Review by sxa — CHANGES_REQUESTED** [review]
Can we add a version check in here since I don't think all versions of Ubuntu on RISC-V have the `docker.io` package available.

**Pull request closed without merging**